documentation/packages/openverse_attribution/index.md

```diff
@@ -0,0 +1,79 @@
+# `openverse-attribution`
```

Comment:
Actually, occurring to me that it'd be important for package README's to be in the package itself, maybe just copied to documentation during the build step, otherwise it isn't available for inclusion in the build output of the package. npm and pypi require a README (`twine check` will fail on build outputs that don't have it, and you won't be allowed to upload it).

Not an issue for this PR, but something we will need to address in the future when we publish packages from the packages directories.